grouper/fe/templates/help.html

```diff
@@ -10,13 +10,90 @@
 
 {% block content %}
 
-<!-- TODO(mildorf): make this more helpful...  -->
+<!-- TODO(mildorf): clean up the style here, it's not 1995. -->
+<!-- TODO(mildorf): keep this in sync with reality. -->
 
 <h3>What is Grouper?</h3>
-<p>This is a group membership and permission management platform. In other words, we are an
-authorization service you can use to answer such questions as: who's in what groups, and what
-permissions do they have?</p>
+<p>Grouper is a group membership and permission management platform. In other words, an
+authorization service you can use to answer such questions as: who's in what groups, and
+what permissions do they have? However, Grouper itself is just a system for permissions
+specfication. It's up to Grouper clients to actually implement things like access
+control.</p>
 
+<h3>How does group membership work?</h3>
+<p>Users and groups are the nodes of a directed acyclic graph.  The users are always leaf
```

Comment:
I'm not sure end users (non-engineers, managers a lot of the time) will know what a DAG is!